sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationLiveTestBase.java

```diff
@@ -51,6 +54,9 @@ public class CallAutomationLiveTestBase extends TestBase {
     protected static final String RECORDING_DELETE_URL_404 = Configuration.getGlobalConfiguration()
         .get("RECORDING_DELETE_URL_404", "https://storage.asm.skype.com/v1/objects/0-eus-d2-3cca2175891f21c6c9a5975a12c0141c");
 
+    protected static final String TARGET_USER_ID = Configuration.getGlobalConfiguration()
```

Comment:
Not needed. The current main branch has this entry already.

Reply:
Please sync up with latest main branch and check "ACS_USER_1"

Reply:
Thanks. I synced your changes and the new Identity client addition is helpful. I ended up creating a new variable for target user in call recording so that I can use the default that I used in RECORD mode. Hope this addresses your comment.